torchani/utils.py

```diff
@@ -2,9 +2,234 @@
 import torch.utils.data
 import math
 import numpy as np
+import time
+import sys
 from collections import defaultdict
 
 
+class Progbar(object):
```

Comment:
Why there is a `Progbar` and a `Progressbar`?

Reply:
Progbar from keras, for training
progressbar from me, for dataloading.